modules/cluster-entitlements.adoc

```diff
@@ -17,33 +17,22 @@
 [id="cluster-entitlements_{context}"]
 = Internet and Telemetry access for {product-title}
 
-In {product-title} {product-version}, Telemetry is the component that provides
-metrics about cluster health and the success of updates. To perform subscription
-management, including legally entitling your purchase from Red Hat, you must use
-the Telemetry service and access the
-link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page.
+In {product-title} {product-version}, Telemetry is the component that provides metrics about cluster health and the success of updates. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is automatically entitled after installation.
 
-Because there is no disconnected subscription management, you cannot both opt
-out of sending data back to Red Hat and entitle your purchase. Support for
-disconnected subscription management might be added in future releases of
-{product-title}
+To perform subscription management, including legally entitling your purchase from Red Hat, you must access the link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers] page. If you cannot send data back to Red Hat, you can perform link:https://cloud.redhat.com/openshift/register[Cluster registration] on this site.
```

Comment:
Nit: "If you cannot send data back to Red Hat" -> "If your cluster cannot..."?  And Maybe drop the "you must access" sentence, or elaborate on any required actions in the cluster-Telemetery-client-is-calling-in case?